selfdrive/car/hyundai/values.py

```diff
@@ -172,6 +173,7 @@ class HyundaiCarInfo(CarInfo):
     HyundaiCarInfo("Kia Sorento 2018", "Advanced Smart Cruise Control", "https://www.youtube.com/watch?v=Fkh3s6WHJz8", harness=Harness.hyundai_c),
     HyundaiCarInfo("Kia Sorento 2019", video_link="https://www.youtube.com/watch?v=Fkh3s6WHJz8", harness=Harness.hyundai_e),
   ],
+  CAR.KIA_SPORTAGE_HYBRID_5TH_GEN: HyundaiCarInfo("Kia Sportage Hybrid 2023", harness=Harness.hyundai_n),
```

Comment:
missing the all supported? the site is confusing since it only mentions that the top trims have navigation SCC, but seems like all have SCC

Reply:
Whoops. Let me fix this.

Lower trims of the Kia Sportages only have SCC for higher trims. Lower trims only get regular Cruise Control.

Reply:
https://www.kiamedia.com/us/en/models/sportage-hev/2023/features
![image](https://user-images.githubusercontent.com/47793918/196063342-94cfbb74-d9b5-49d5-98c7-588322c539f1.png)